deps/sqlite3.gyp

```diff
@@ -51,8 +55,10 @@
         ],
         'link_settings': {
           'libraries': [
-            '-llibeay32.lib',
-            '-lssleay32.lib',
+            '-llibcrypto.lib',
+            '-llibssl.lib',
+            '-lws2_32.lib',
+            '-lcrypt32.lib'
```

Comment:
Are these two required? Would be good to have a comment on what it's used for / why it's needed.

Reply:
Not required indeed. Came across them in https://github.com/EvanHahn-Signal/node-sqlcipher/commit/16916949f0c010f6e6d3d5869b10a0ab813eae75 so also added them here, but just tested without and it build/tests without problems 😊

Reply:
Actually, they _are_ needed for the Electron build to succeed. Otherwise it fails with the error below. Just added a note about that in https://github.com/journeyapps/node-sqlcipher/pull/75/commits/35ac248ccc9608132a89f30761efe8b073fe1a31

```
Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
  nothing.c
  win_delay_load_hook.cc
  nothing.vcxproj -> F:\repos\node-sqlcipher\build\Release\\nothing.lib
  unpack_sqlite_dep
  Copying F:\repos\node-sqlcipher\build\Release\obj/global_intermediate/sqlcipher-amalgamation-3033000/OpenSSL-Win64/libcrypto.lib to F:\repos\node-sqlcipher\build\Release\\libcrypto.lib
          1 file(s) copied.
  Copying F:\repos\node-sqlcipher\build\Release\obj/global_intermediate/sqlcipher-amalgamation-3033000/OpenSSL-Win64/libssl.lib to F:\repos\node-sqlcipher\build\Release\\libssl.lib
          1 file(s) copied.
  Copying F:\repos\node-sqlcipher\build\Release\obj/global_intermediate/sqlcipher-amalgamation-3033000/OpenSSL-Win64/ossl_static.pdb to F:\repos\node-sqlcipher\build\Release\\ossl_static.pdb
          1 file(s) copied.
  sqlite3.c
  win_delay_load_hook.cc
  sqlite3.vcxproj -> F:\repos\node-sqlcipher\build\Release\\sqlite3.lib
  backup.cc
  database.cc
  node_sqlite3.cc
  statement.cc
  win_delay_load_hook.cc
     Creating library F:\repos\node-sqlcipher\build\Release\node_sqlite3.lib and object F:\repos\node-sqlcipher\build\Release\node_sqlite3.exp
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_ioctlsocket [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_getsockname [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_getsockopt [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_getsockopt [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_ntohs [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_addr.obj) : error LNK2001: unresolved external symbol __imp_ntohs [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_gethostbyname [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_WSAStartup [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_WSACleanup [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock.obj) : error LNK2001: unresolved external symbol __imp_WSAGetLastError [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(bss_sock.obj) : error LNK2001: unresolved external symbol __imp_WSAGetLastError [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_WSAGetLastError [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_addr.obj) : error LNK2001: unresolved external symbol __imp_getaddrinfo [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_addr.obj) : error LNK2001: unresolved external symbol __imp_freeaddrinfo [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_addr.obj) : error LNK2001: unresolved external symbol __imp_getnameinfo [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertOpenStore [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertCloseStore [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertEnumCertificatesInStore [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertFindCertificateInStore [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertDuplicateCertificateContext [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertFreeCertificateContext [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(e_capi.obj) : error LNK2001: unresolved external symbol __imp_CertGetCertificateContextProperty [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(bss_sock.obj) : error LNK2001: unresolved external symbol __imp_recv [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(bss_sock.obj) : error LNK2001: unresolved external symbol __imp_send [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(bss_sock.obj) : error LNK2001: unresolved external symbol __imp_WSASetLastError [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_accept [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_bind [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_closesocket [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_connect [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_listen [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_setsockopt [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
libcrypto.lib(b_sock2.obj) : error LNK2001: unresolved external symbol __imp_socket [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
F:\repos\node-sqlcipher\build\Release\node_sqlite3.node : fatal error LNK1120: 28 unresolved externals [F:\repos\node-sqlcipher\build\node_sqlite3.vcxproj]
```